ethers-core/src/types/trace/filter.rs

```diff
@@ -266,6 +266,32 @@ pub enum RewardType {
     External,
 }
 
+/// Basic trace type for the Geth client.
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct GethTrace {
+    failed: bool,
+    gas: i64,
```

Comment:
lets make gas `U256`, that's consistent across other ethers types

Reply:
Done.